informer/collector.go

```diff
@@ -0,0 +1,65 @@
+package informer
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"k8s.io/apimachinery/pkg/api/meta"
+)
+
+var (
+	description *prometheus.Desc = prometheus.NewDesc(
+		prometheus.BuildFQName("operatorkit", "informer", "deletion_timestamp"),
+		"DeletionTimestamp of watched objects.",
+		[]string{
+			"name",
+			"namespace",
```

Comment:
What about the operator name actually? Isn't this important/useful?

Reply:
We get this through the `app` label by prometheus already, no need to add it here.